<launch>
    <node pkg="robot_interface" type="robot_interface_node" name="robot_interface" output="screen">
        <!-- set a private parameter for the node -->
        <param name="goalTopic" type="string" value="cmd"/>
        <param name="subTopic" type="string" value="cmd_vel"/>
        <param name="serialName" type="string" value="/dev/CBoard"/>
        <param name="fixFrame" type="string" value="world"/>
        <param name="robotFrame" type="string" value="base"/>
        <param name="gimbalFrame" type="string" value="gimbal"/>
        <param name="base2gimbal_x" type="double" value="0"/>
        <param name="base2gimbal_y" type="double" value="0"/>
        <param name="base2gimbal_z" type="double" value="0.5"/>
        <param name="init_x" type="double" value="8.8"/>
        <param name="init_y" type="double" value="8.8"/>
        <!-- BLUE is 1, RED is 0 -->
        <param name="preset_side" type="int" value="0"/>
    </node>

</launch>
